openvdb/CHANGES

```diff
@@ -12,6 +12,8 @@ Version 6.1.1 - In development
       now modify voxel values in place for improved performance.
     - Add CMake support for linking against Jemalloc and TBB malloc and enable
       Jemalloc by default for Linux and non-Maya, otherwise use TBB malloc.
+    - Made a few improvements related to the use of std::inf and std::nan with
```

Comment:
How about "Added math::isInfinite() and math::isNaN() to resolve Visual Studio compatibility issues with integer types"?

Reply:
Sure